walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,80 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.ArrayList;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
```

Comment:
I forgot to mention that this is not the most optimal solution. This is `O(e^2 + e*p)` but can be reduced to `O(e*log(e) + e*p)` where `e=number of events` and `p=number of participants (optional + mandatory)`. I encourage you to think about how this algorithm could be improved.